docker/deploy-scripts/zms-deploy.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
+
+set -eu
+set -o pipefail
+
+# to script directory
+cd "$(dirname "$0")"
+
+# import functions
+source ../setup-scripts/common/color-print.sh
```

Comment:
I understand why it fails on ubuntu.
https://stackoverflow.com/questions/13702425/source-command-not-found-in-sh-shell

For the time being, I think it's okay to exclude ubuntu from support, but what do you think?

Reply:
sure, can you help to fix the script?

Reply:
since it is difficult to be compatible with `dash` in Ubuntu, added note in a8740c7e77447d5e349a720d07660e86d1089f22.